src/common/global-hot-key/global-hot-key.ts

```diff
@@ -3,5 +3,6 @@ import { GlobalHotKeyKey } from "./global-hot-key-key";
 
 export interface GlobalHotKey {
     key: GlobalHotKeyKey;
-    modifier?: GlobalHotKeyModifier;
+    firstModifier?: GlobalHotKeyModifier;
```

Comment:
If we wouldn't rename `modifier` to `firstModifier` we wouldn't need any migration. Why don't we just add `secondModifier` ?

Reply:
Alright, fair enough. I've renamed `firstModifier` and added small fallback logic for missing `secondModifier`.